api/queries_sponsors.go

```diff
@@ -0,0 +1,90 @@
+package api
+
+import "net/http"
+
+type SponsorQuery struct {
+	User User `json:"user" graphql:"user(login: $username)"`
+}
+
+type User struct {
+	Sponsors Sponsors `json:"sponsors" graphql:"sponsors(first: 30)"`
+}
+
+type Sponsors struct {
+	TotalCount int    `json:"totalCount"`
+	Nodes      []Node `json:"nodes"`
+}
+
+type Node struct {
+	Login string `json:"login"`
+}
+
+func GetSponsorsList(httpClient *http.Client, hostname string, username string) ([]string, error) {
+	client := NewClientFromHTTP(httpClient)
+	//queryResult, err := querySponsorsViaReflection(client, hostname, username)
+	queryResult, err := querySponsorsViaStringManipulation(client, hostname, username)
+	if err != nil {
+		return nil, err
+	}
+
+	return mapQueryToSponsorsList(queryResult), nil
+}
+
+// This is currently broken. I suspect the issue is that the SponsorQuery
+// struct is not defining the variable $username correctly. I need to go
+// find an example query with a variable passed in like this to see how its
+// done. In the mean time, I'm going to move forward with the string
+// manipulation version of this function.
+func querySponsorsViaReflection(client *Client, hostname string, username string) (SponsorQuery, error) {
+	query := SponsorQuery{}
+	variables := map[string]interface{}{
+		"username": username,
```

Comment:
```suggestion
                "username": githubv4.String(username),
```

I think this is probably going to fix your error but since I don't know exactly what it is I'm not sure.